scripts/build_release_centos_docker.sh

```diff
@@ -64,7 +60,10 @@ rm -f ${HOME}/.gnupg/S.gpg-agent
 (cd ~/.gnupg && ln -s /S.gpg-agent S.gpg-agent)
 
 gpg --import /stuff/key.pub
-gpg --import ${REPO_ROOT}/installer/rpm/RPM-GPG-KEY-Algorand
+gpg --import /stuff/rpm.pub
+gpg --import ${REPO_DIR}/installer/rpm/RPM-GPG-KEY-Algorand
+rpmkeys --import /stuff/rpm.pub
+echo "wat"|gpg -u rpm@algorand.com --clearsign
```

Comment:
Out of curiosity, what is this command for?

Reply:
It's a quick check to make sure signing works at all, before doing some stuff that takes more time. it's a fail-fast check. It was useful for debugging and I don't think there's any reason to take it out. GPG is a little fragile, so leaving it in is probably good in case it catches things in the future.